compiled_starters/gleam/src/sqlite.gleam

```diff
@@ -0,0 +1,26 @@
+import argv
+import file_streams/read_stream
+import gleam/bit_array
+import gleam/int.{to_string}
+import gleam/io
+import gleam/list.{each}
+
+pub fn main() {
+  // You can use print statements as follows for debugging, they'll be visible when running tests.
+  io.println("Logs from your program will appear here!")
+  let args = argv.load().arguments
+  // Uncomment this to pass the first stage
+  // case args {
+  //   [database_file_path, ".dbinfo", ..] -> {
+  //     let assert Ok(rs) = read_stream.open(database_file_path)
+  //     let assert Ok(bytes) = read_stream.read_bytes_exact(rs, 16)
```

Comment:
@ryan-gang let's add comments on what this is doing. Alternatively, if asserts with values are supported, can we do something like `assert OK("SQLite format 3\000") = read_stream...`?